heareval/predictions/task_predictions.py

```diff
@@ -360,8 +360,8 @@ def __init__(
         nlabels: int,
         prediction_type: str,
         scores: List[ScoreFunction],
-        validation_target_events: Dict[str, List[Dict[str, Any]]],
-        test_target_events: Dict[str, List[Dict[str, Any]]],
+        validation_target_events: Union[Dict[str, List[Dict[str, Any]]], None],
```

Comment:
Wait why is the signature of this changing? I'm confused

Reply:
The addition of None here and below is confusing to me

Reply:
Ya I think this is a bug.